src/core/attribution/markovChain.js

```diff
@@ -102,12 +113,15 @@ function* findStationaryDistributionGenerator(
   chain: SparseMarkovChain,
   options: {|
     +verbose: boolean,
+    // The chain is considered converged once the maximum delta in the
```

Comment:
This is at best unclear. What does “maximum delta in the distribution”
mean? It suggests “the largest difference between any two components in
the distribution”, but of course that is not what is meant.

Suggest: “A distribution is considered stationary if the action of the
Markov chain on the distribution does not change any component by more
than `convergenceThreshold` in absolute value.”

Reply:
Done.